src/main/java/org/broadinstitute/hellbender/tools/copynumber/utils/MergeAnnotatedRegions.java

```diff
@@ -0,0 +1,64 @@
+package org.broadinstitute.hellbender.tools.copynumber.utils;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.BetaFeature;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.ExomeStandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.CopyNumberProgramGroup;
+import org.broadinstitute.hellbender.engine.GATKTool;
+import org.broadinstitute.hellbender.tools.copynumber.utils.annotatedregion.SimpleAnnotatedGenomicRegion;
+import org.broadinstitute.hellbender.tools.copynumber.utils.annotatedregion.SimpleAnnotatedGenomicRegionCollection;
+import org.broadinstitute.hellbender.tools.copynumber.utils.annotatedregion.SimpleAnnotatedGenomicRegionUtils;
+import org.broadinstitute.hellbender.utils.IntervalUtils;
+
+import java.io.File;
+import java.util.List;
+
+@CommandLineProgramProperties(
+        oneLineSummary = "(EXPERIMENTAL) Merge annotated genomic regions based entirely on contig and annotation value.",
+        summary = "(EXPERIMENTAL) Merge annotated genomic regions based entirely on contig and annotation value.  Column header order is not guaranteed to be preserved. Reference is required and will superseded any sequence dictionary in the given seg/region files.  Sequence dictionary is optional on the input file, but will be included on the output.  Conflicts of annotations is resolved by sorting the values and inserting a delimiter.  THIS TOOL IS TOTALLY UNSUPPORTED",
+        programGroup = CopyNumberProgramGroup.class)
+
+@BetaFeature
+public class MergeAnnotatedRegions extends GATKTool {
+    @Argument(
+            doc = "Input segment file or annotated segment file -- touching segments will be merged in the output.",
+            fullName = ExomeStandardArgumentDefinitions.SEGMENT_FILE_LONG_NAME,
```

Comment:
Oops, guess I forgot to purge `ExomeStandardArgumentDefinitions`.  Can you delete and switch over to `CopyNumberStandardArgument`?

Reply:
Done.  No short name there?  Seems like segment files could have a short name